openpype/hosts/maya/tools/mayalookassigner/vray_proxies.py

```diff
@@ -263,16 +180,16 @@ def vrayproxy_assign_look(vrayproxy, subset="lookDefault"):
             ))
             continue
 
-        relationships = get_look_relationships(version["_id"])
-        shadernodes = load_look(version["_id"])
+        relationships = lib.get_look_relationships(version["_id"])
+        shadernodes, _ = lib.load_look(version["_id"])
 
         # Get only the node ids and paths related to this asset
         # And get the shader edits the look supplies
         asset_nodes_by_id = {
             node_id: nodes_by_id[node_id] for node_id in node_ids
         }
         edits = list(
-            lib.iter_shader_edits(
+            api.lib.iter_shader_edits(
```

Comment:
Question: Why would we want to directly turn this into a list instead of just iterating over it in the for loop?
Code is likely based on existing code - but just wondering.